### ISSUE TITLE
UnparsedTextResolver seems to be broken for Saxon 11+

### ISSUE THREAD

**Issue by nkutsche** [ndw/xmlcalabash1#346]
There seems a general problem with the UnparstTextResolver for the editions `110` and `120`.

In any case using the `unparste-text()` function I get with `1.5.6-110`:

```
ERROR com.xmlcalabash.drivers.Main - Pipeline failed: Resolver for unparsed-text() returned unrecognized Source (org.xmlresolver.sources.ResolverSAXSource)
ERROR com.xmlcalabash.drivers.Main - Underlying exception: Resolver for unparsed-text() returned unrecognized Source (org.xmlresolver.sources.ResolverSAXSource)
```

Or with `1.5.6-120` a slightly different error message:

```
ERROR com.xmlcalabash.drivers.Main - Pipeline failed: Resolver for unparsed-text() returned non-StreamSource
ERROR com.xmlcalabash.drivers.Main - Underlying exception: Resolver for unparsed-text() returned non-StreamSource
```

With `1.5.6-100` it works as expected (ignoring the problems described in #345).

**Comment by ndw** [comment]
Thanks. There were some Saxon issues in this area too. I'll try to take a look this weekend.

**Comment by palimondo** [comment]
I have the same issue with 1.5.6-120, it comes up when using `json-doc` function. I have temporarily switched back to 1.5.3.-110 which does not have this issue, so it seems to be related to upgrade from Saxon 11 to 12.

**Comment by ndw** [comment]
I think this is fixed by the same patches I added for #345

**Comment by ndw** [comment]
Please reopen this issue if it isn't resolved in 1.5.7
